src/Collection/CollectionTrait.php

```
@@ -702,7 +702,7 @@ public function isEmpty()
     public function unwrap()
     {
         $iterator = $this;
-        while (get_class($iterator) === 'Cake\Collection\Collection') {
+        while ($iterator instanceof Collection) {
```

Okay, I'll revert this and ignore the error instead.